compute/ARMComputeEx/src/runtime/CL/functions/CLArgMinMaxLayerEx.cpp

```diff
@@ -0,0 +1,205 @@
+/*
```

Comment:
please add copyright

Reply:
Added to all modified files under armcomputeex, kindly review. Thanks.